core/src/main/java/io/atomix/core/multimap/ConsistentMultimapType.java

```diff
@@ -48,9 +55,20 @@ public String name() {
     return NAME;
   }
 
+  @Override
+  public Namespace namespace() {
```

Comment:
HashMultiset.create()
ImmutableSet.of()
ImmutableList

classes defined in super namespace?

Reply:
yep